### ISSUE TITLE
Remove deprecated t.a.app.ApplicationRunner.startLogging

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4092]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#4092|
|Type|task|
|Created|2009-11-13 19:20:34Z|

```ApplicationRunner.startLogging``` in [twisted/application/app.py](twisted/application/app.py) has been deprecated (doesn't tell since when), time to get rid of it (eventually).

Attachments:

* [4092.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/c95/c95ac6471fe1bd8d916a8ec79e4d14a108769a75/c51de1f2001b298d69664ca1cebccf9410ed7601.diff) (3431 bytes) - added by cyli on 2010-05-23 23:33:14Z - Removal of startLogging and getLogObserver in AppRunner

<details><summary>Searchable metadata</summary>

```
trac-id__4092 4092
type__task task
reporter__thijs thijs
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__easy easy
time__1258140034000000 1258140034000000
changetime__1275247103000000 1275247103000000
version__None None
owner__ 
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)|@glyph removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_2"></a>|@cyli commented|
|-|-|

Also removed the getLogObserver legacy test in test_twistd.py, since it seems like it's no longer applicable to ApplicationRunner

**Comment by twisted-trac** [comment]
|[<img alt="mithrandi's avatar" src="https://avatars.githubusercontent.com/u/856430?s=50" width="50" height="50">](https://github.com/mithrandi)<a name="note_3"></a>|@mithrandi set owner to @cyli|
|-|-|

Looks good. However, I think 4092.removal should also mention the removal of support for using a getLogObserver method; please fix this before merging.

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_4"></a>|@cyli set status to `closed`|
|-|-|

(In [29105]) Applied patch 4092.diff from ticket 4092: remove ApplicationRunner.startLogging

Author: cyli 
Reviewer: mithrandi 
Fixes: [#4092](https://github.com/twisted/twisted/issues/4092)

Removed twisted.application.app.ApplicationRunner.startLogging, which has been deprecated, and support for the legacy twisted.application.app.ApplicationRunner.getLogObserver method.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
